src/main/java/seedu/nutrigoals/model/ModelManager.java

```diff
@@ -118,7 +123,8 @@ public void deleteFood(Food target) {
     @Override
     public void addFood(Food food) {
         nutriGoals.addFood(food);
-        updateFilteredFoodList(new IsFoodAddedTodayPredicate());
+        currentDatePredicate = new IsFoodAddedOnThisDatePredicate(new DateTime());
```

Comment:
Would it work if only `updateFilteredFoodList(new IsFoodAddedOnThisDatePredicate(new DateTime())` is called?

Reply:
Yes, I didn't notice that I was assigning `currentDatePredicate` twice! I'll make the changes!